packages/react-core/src/demos/examples/Tabs/ModalTabs.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import DashboardWrapper from '../DashboardWrapper';
+import {
+  PageSection,
+  TextContent,
+  Text,
+  Gallery,
+  Card,
+  CardBody,
+  CardTitle,
+  PageSectionVariants,
+  Modal,
+  ModalVariant,
+  Tab,
+  Tabs,
+  TabTitleText,
+  List,
+  ListItem,
+  Grid,
+  GridItem,
+  TabContent
+} from '@patternfly/react-core';
+
+interface Product {
+  id: number;
+  name: string;
+  description: string;
+}
+
+const products: Product[] = [
+  {
+    id: 0,
+    name: 'PatternFly',
+    description: 'PatternFly is a community project that promotes design commonality and improves user experience.'
+  },
+  {
+    id: 1,
+    name: 'ActiveMQ',
+    description:
+      'The ActiveMQ component allows messages to be sent to a JMS Queue or Topic; or messages to be consumed from a JMS Queue or Topic using Apache ActiveMQ.'
+  },
+  {
+    id: 2,
+    name: 'Apache Spark',
+    description: 'This documentation page covers the Apache Spark component for the Apache Camel.'
+  }
+];
+
+export const ModalTabs: React.FunctionComponent = () => {
+  const [isModalOpen, setIsModalOpen] = React.useState(false);
```

Comment:
This default value infers what the type is, are we sure we want to add explicit types when types are inferred?

If we do want to mention the type regardless, I assume we would only be doing this because it's an example?